libcodechecker/util.py

```diff
@@ -570,13 +570,21 @@ def load_json_or_empty(path, default=None, kind=None):
     ret = default
     try:
         with open(path, 'r') as handle:
+            if lock:
+                portalocker.lock(handle, portalocker.LOCK_SH)
+
             ret = json.loads(handle.read())
-    except IOError:
+
+            if lock:
+                portalocker.unlock(handle)
+    except IOError as ex:
         LOG.warning("Failed to open {0} file: {1}"
                     .format(kind if kind else 'json', path))
-    except ValueError:
+        LOG.warning(ex)
+    except ValueError as ex:
```

Comment:
Can we get `OSError`, `TypeError` or some other exception which should be handled here?